### PR TITLE
[Dojot-module-nodejs] Remove travis and make changes to use github actions

### PR DESCRIPTION
**Pull request by marianoleonardo**
dojot/dojot#1913

### CODE REVIEW

**Review by mprevide — CHANGES_REQUESTED** [review]
There are 2 points
 - documentation
 - auto publish in npm

**Comment by mprevide** [comment]
Can you publish in your npm account?

**Review by mprevide — APPROVED** [review]
Don't worry about DeepScan error

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+jobs:
+
+  integration-tests:
+    name: Integration tests
+    # If the commit message contains 'ci skip', it is not necessary to run this job
+    if: contains(github.event.head_commit.message, 'ci skip') == false
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        node-version: [8.x]
```
Comment (thiagodpf) [inline .github/workflows/main.yml:18]: This module is now available for Node.js v12.x

File: .github/workflows/publish-npm-packages.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          run: |
+            npm ci
+            npm run build --if-present
+            npm test
+            npm run check-licenses
+
+        - name: Set registry to npmjs.com
+          uses: actions/setup-node@v1
+          with:
+            node-version: '10.x'
```
Comment (thiagodpf) [inline .github/workflows/publish-npm-packages.yml:46]: you may also have to switch to version `12.x` here...

Comment (thiagodpf) [reply .github/workflows/publish-npm-packages.yml:46]:
```waml
with:
          node-version: ${{ matrix.node-version }}
```

Comment (mprevide) [reply .github/workflows/publish-npm-packages.yml:46]: yes, but almost all services that use it are in node 8 or 10

File: .github/workflows/main.yml (hunk 1 above)
Comment (mprevide) [reply .github/workflows/main.yml:18]: yes, but almost all services that use it are in node 8 or 10

Comment (mprevide) [reply .github/workflows/main.yml:18]: maybe just here you must change to `[8.x, 10.x, 12.x]`

**Review by thiagodpf — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (mprevide) [reply .github/workflows/main.yml:18]: Maybe you can change to 10.x

**Review by mprevide — APPROVED** [review]

**Review by mprevide — CHANGES_REQUESTED** [review]
Remove badge travis from readme

**Review by mprevide — APPROVED** [review]

**Pull request merged**